app/(landing)/elections/position/[...loc]/components/Hero.js

```diff
@@ -66,7 +67,16 @@ export default function Hero({
         </div>
       </div>
       <div className="pb-5">
-        <div className="flex justify-between items-end">
+        <div className={
+          classNames(
+            'flex',
+            'justify-between',
+            'md:items-end',
+            'md:flex-row',
+            'flex-col',
+            'items-start'
+          )
+        }>
```

Comment:
Please keep the same style we have. no need to introduce classnames unless you have dynamic classes that can overwrite eachother

Reply:
Addressed both here: https://github.com/thegoodparty/gp-webapp/pull/116/commits/efb00788503d8a99b97005a86ee86b8fe2ada3cf